source/grid/tria.cc

```diff
@@ -15786,10 +16006,14 @@ void Triangulation<dim, spacedim>::execute_coarsening_and_refinement()
   // Inform all listeners about beginning of refinement.
   signals.pre_refinement();
 
+  this->pack_data_serial();
+
   execute_coarsening();
 
   const DistortedCellList cells_with_distorted_children = execute_refinement();
 
+  this->unpack_data_serial();
```

Comment:
This seems to support my suspicion that the functions mentioned above are perhaps not named in a way that reveals what they do.